extensions/schema-compare/src/schemaCompareMainWindow.ts

```diff
@@ -131,7 +131,7 @@ export class SchemaCompareMainWindow {
 				projectFilePath: '',
 				targetScripts: [],
 				dataSchemaProvider: '',
-				folderStructure: ''
+				folderStructure: mssql.ExtractTarget.none
```

Comment:
Not something to fix now, but these properties that aren't common across the different schema compare endpoint types should really be optional so they don't need to be set to empty strings here

Reply:
Yeah that makes sense. I will create a workitem for it to keep track.

Reply:
Created https://github.com/microsoft/azuredatastudio/issues/21411